songData.js

```diff
@@ -22,31 +22,47 @@ const songDataSchema = new mongoose.Schema({
 
 const Song = mongoose.model('Song', songDataSchema);
 
-var saveSong = (songData, callback) => {
-  let song = new Song(songData);
-  song.save((err, data) => {
-    if (err) {
-      console.log('Error in saving song to db');
-      callback(err);
-    } else {
-      console.log(`Song, "${song.songName}" saved to DB`);
-      callback(null, data);
-    }
-  })
+var saveSong = async (songData, callback) => {
+  try {
+    let song = new Song(songData);
+    var saved = await song.save(( ) => {
```

Comment:
await in the try, good!

But wait? why are you returning to the saved variable? you're not sending it back. How would you send it back? would you want to? 

Are you going to use the callback?